esx_service/vmci/connection_errors.h

```diff
@@ -0,0 +1,87 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Errors on vmci command communication channel
+
+#ifndef _CONNECTION_ERRORS_H_
+#define _CONNECTION_ERRORS_H_
+
+typedef enum VMCI_ConnectionError {
+   CONN_SUCCESS = 0,
+
+   // Misc. protocol errors. When changing, also update info[] below
+   CONN_FAILED_VMCI_ADDRESS_FAMILY_GET,
```

Comment:
The error should reflect the error vs. the function that failed. The original was correct - CONN_NO_VMCI_ADDRESS_FAMILY., New one can be CONN_FAILED_NO_VMCI_ADDRESS_FAMILY.